src/msg/async/dpdk/DPDK.cc

```diff
@@ -531,6 +532,7 @@ bool DPDKQueuePair::init_rx_mbuf_pool()
     for (int i = 0; i < mbufs_per_queue_rx; i++) {
       rte_mbuf* m = rte_pktmbuf_alloc(_pktmbuf_pool_rx);
       assert(m);
+      rte_mbuf_refcnt_update(m, -1);
```

Comment:
?

Reply:
if not clear refcnt here, will cause assert 2